src/pages/home/index.jsx

```diff
@@ -106,21 +106,24 @@ export function App() {
               <Input
                 borderNone={true}
                 type="text"
-                placeholder="pesquisar produtos"
+                placeholder="Pesquisar produtos"
                 onChange={(event) => {
                   setSearch(event.currentTarget.value);
                 }}
               />
             </div>
           </div>
           <div className={style["button-icons"]}>
+
+            <div className={style["div-button"]}>
```

Comment:
It's not indented here, talk to @isacmoreira10 to give you a guide that will indent the code on its own